src/auth/jwtclient.ts

```diff
@@ -14,48 +14,24 @@
  * limitations under the License.
  */
 
-const gToken = require('gtoken');
+import {GoogleToken, TokenOptions} from 'gtoken';
+import * as request from 'request';
+import * as stream from 'stream';
+
 import {Credentials, JWTInput} from './credentials';
 import {JWTAccess} from './jwtaccess';
 import {OAuth2Client} from './oauth2client';
-import * as stream from 'stream';
-import * as request from 'request';
 
 const isString = require('lodash.isstring');
 const noop = Function.prototype;
 
-export interface TokenOptions {
-  key?: string|null;
-  iss?: string|null;
-  sub?: string|null;
-  scope?: string|string[]|null;
-  keyFile?: string|null;
-  // email: string;
-}
-
-export declare interface GoogleToken {
-  (opts: TokenOptions): GoogleToken;
-  getToken(callback: (err?: Error|null, token?: string|null) => void): void;
-  token: string|null;
-  expires_at: number|null;
-  key: string|undefined;
-  keyFile: string|undefined;
-  iss: string|undefined;
-  sub: string;
-  scope: string|undefined;
-  rawToken: string|null;
-  tokenExpires: number|null;
-  email: string;
-}
-
 export class JWT extends OAuth2Client {
   email?: string;
   keyFile?: string|null;
   key?: string|null;
   scopes?: string|string[]|null;
   scope?: string|null;
   subject?: string;
-  gToken: GoogleToken;
```

Comment:
semver major?

Reply:
So this was super weird.  It was exposing a reference to the GoogleToken.getToken function, which was only public so it could be mocked in tests.  While technically yes this is a breaking change, I can't imagine anyone is using it this way.  

That having been said - yes, lets lump this in with the other breaking changes we have for the 1.0 release.